src/lib/y2storage/crypttab.rb

```diff
@@ -31,14 +31,19 @@ class Crypttab
     CRYPTTAB_PATH = "/etc/crypttab"
     private_constant :CRYPTTAB_PATH
 
+    # @return [Filesystems::Base]
```

Comment:
NP: documentation. To know what `#filesystem` is really about I need to check the documentation of the constructor, which is not obvious from here.

I would document here what the attribute is about, not in the constructor.